src/ast-utils/element-finder.ts

```diff
@@ -40,8 +40,8 @@ export function elementFinder() {
       this.classDetected = true;
       element = {
         type: 'element',
-        desc: esutil.getAttachedComment(parent),
-        events: esutil.getEventComments(parent).map(function(event) {
+        desc: esutil.getAttachedComment(node),
```

Comment:
Class support - the parent node of the class declaration is the Program, so all events were getting associated with every es6 class in the file.